app/entities/transfer.go

```diff
@@ -0,0 +1,60 @@
+package entities
+
+import (
+	"errors"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Transfer struct {
+	transferID           string
```

Comment:
Não precisa mudar, mas daria pra simplificar para apenas `id`, já q vc deduz q eh o id da entidade transfer
```suggestion
	id          string
```